bin/warnhist

```diff
@@ -185,6 +371,20 @@ if($options{u}) {
     $genericize = 0;
 }
 
+# Scan source files, if warning density requested
```

Comment:
Scan the source file paths if warning density is requested.  (Note that '.' is used if no source file paths are given)